doc/classes/EditorSettings.xml

```diff
@@ -784,13 +784,15 @@
 			If set to [code]Adaptive[/code], the dialog opens in list view or grid view depending on the requested type. If set to [code]Last Used[/code], the display mode will always open the way you last used it.
 		</member>
 		<member name="filesystem/quick_open_dialog/enable_fuzzy_matching" type="bool" setter="" getter="">
-			If [code]true[/code], fuzzy matching of search tokens is allowed.
+			If [code]true[/code], together with exact matches of a filename, the dialog includes approximate matches.
+			This is useful for finding the correct files even when there are typos in the search query, e.g., searching "nprmal" will find "normal". Additionally, it allows you to write shorter search queries, e.g., searching "nml" will also find "normal".
```

Comment:
```suggestion
			This is useful for finding the correct files even when there are typos in the search query, for example, searching "nprmal" will find "normal". Additionally, it allows you to write shorter search queries, for example, searching "nml" will also find "normal".
```
Consistency with abbreviations

Reply:
Sounds good to me, although I think the preceding commas will need to be replaced with semicolons to avoid being run-on sentences